pages/configuration.tsx

```diff
@@ -181,6 +181,7 @@ function Instructions() {
           `WEBHOOK_URL` field
         </ListItem>
         <ListItem>Save configuration</ListItem>
+        <ListItem>Create new order to test application</ListItem>
```

Comment:
Hm I think it can be done better. This text is in "configure" section and its quite hidden

I was thinking that maybe the form input with wwebhook URL can have some label or helper text "this webhook will be called when new order is created (ORDER_CREATED))" ?